.github/workflows/judoscale-rails-test.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       matrix:
         gemfile:
           - Gemfile
+          - Gemfile-rails-7-1
```

Comment:
Adding a new Gemfile to test Rails 7.1, keeping the main Gemfile testing 7.2 (8.0 still in alpha)